docs/en/Running-Kylin-on-Alluxio.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: Running Apache Kylin on Alluxio
+nickname: Apache Kylin
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache Kylin](http://kylin.apache.org/) on Alluxio, so
+that you can speed up the cube build phase in Kylin.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](Java-Setup.html). Alluxio cluster should also be
+set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
+[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please [Download Kylin](http://kylin.apache.org/download/).
+
+# Configuration
+
+Kylin requires a setup environment including [Hive](http://hive.apache.org/),
+[Hadoop](http://hadoop.apache.org/) and [HBase](http://hbase.apache.org/). You
+need to refer the following documentations for how to run these frameworks on Alluxio:
+
+ [Running Hadoop MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html)
+ 
+ [Running Apache HBase on Alluxio](Running-HBase-on-Alluxio.html)
+ 
+ [Running Apache Hive with Alluxio](Running-Hive-with-Alluxio.html)
+
+#### Set property in Hadoop `core-site.xml`
+
+You need to change the following property in `core-site.xml` in your Hadoop installation
+`etc/hadoop` directory, which ensures that Kylin's Hadoop engine uses Alluxio as its default
+filesystem.
+
+```xml
+<property>
+   <name>fs.defaultFS</name>
```

Comment:
Is there a way to avoid setting the defaultFS and just use the alluxio scheme? Setting defaultFS might be restrictive in certain environments.